### ISSUE TITLE
Unable to retrieve a list of cookies.

### ISSUE THREAD

**Issue by JoshGlazebrook** [johntitus/node-horseman#31]
So I have tried the following code on node 0.10.x, 0.12.x and iojs 2.3.x with phantomjs 1.9.8 and 2.0.0 on OSX Yosemite. All I'm getting is an error that getCookies is not a function on the phantom object.

```
TypeError: this.phantom.getCookies is not a function
    at Horseman.exports.cookies (/Users/Josh/Desktop/Projects/horse-test/node_modules/node-horseman/lib/actions.js:233:18)
    at Object.<anonymous> (/Users/Josh/Desktop...
```

 ```javascript
var Horseman = require('node-horseman');

var horseman = new Horseman({
 cookiesFile: 'cookies.txt'
 });

var result = horseman
    .open('http://google.com')
    .cookies();

console.log(result);
```

I know it's getting the cookies because it's saving to cookies.txt that I set in the options object.

Any idea what's going on?

**Comment by johntitus** [comment]
Sorry, was out of town. I don't know what's going on off hand, but I'll take a look when I get a chance.

**Comment by johntitus** [comment]
Duplicate of #27.

**Comment by johntitus** [comment]
Fixed in 1.5.3.
